poker.js

```diff
@@ -77,44 +81,44 @@ const isStraightFlush = (hand) => {
   return (suitHand.find(num => num === 5) && isStraight === 4 || (suitHand.find(num => num === 5) && rankArr.find(num => num === '5') && rankArr.find(num => num === '4') && rankArr.find(num => num === '3') && rankArr.find(num => num === '2') && rankArr.find(num => num === 14)));
 };
 
-const isFourOfaKind = (hand) => {
-  const rankHand = getValues(rank(hand));
-  return (rankHand.find(num => num === 4));
-};
-
-const isFullHouse = (hand) => {
-  const rankHand = getValues(rank(hand));
-  return (rankHand.find(num => num === 3) && rankHand.find(num => num === 2) && rankHand.length === 2);
-};
-
 const isFlush = (hand) => {
   const suitHand = getValues(suit(hand));
   return (suitHand.find(num => num === 5));
 };
 
 const isStraight = (hand) => {
   const isStraight = straight(hand);
-  return (isStraight === 4);
-};
-
-const isThreeOfaKind = (hand) => {
-  const rankHand = getValues(rank(hand));
-  return (rankHand.find(num => num === 3) && rankHand.length === 3);
+  return (isStraight);
 };
 
-const isTwoPair = (hand) => {
-  const rankHand = getValues(rank(hand));
-  return (rankHand.find(num => num === 2) && rankHand.length === 3);
-};
-
-const isOnePair = (hand) => {
-  const rankHand = getValues(rank(hand));
-  return (rankHand.find(num => num === 2));
-};
-
-const isHighCard = (hand) => {
+const fullOfAKindAndPairs = (hand) => {
```

Comment:
This is not quite the purpose of a function to do so many things. The point is to use reusuable logic with smaller functions. We can clarify that together though